addons/intuit-quickbooks/template/index.js

```diff
@@ -0,0 +1,77 @@
+const qbo = require('./quickbooks.js');
+const health = require('./health.js');
+const util = require('util');
+const Superagent = require('superagent');
+const Sdk = require('@fusebit/add-on-sdk');
+const QuickBooks = require('node-quickbooks');
+
+let client;
+
+module.exports = (ctx, callback) => {
+  if (client) {
+    const TOKEN_EXPIRY_BUFFER = 120;
```

Comment:
Put this around line 8 so that it's separated from the execution code, and collected (if there were any) with other constants at the top of the file.

Reply:
Fixed